CONTRIBUTING.md

```diff
@@ -0,0 +1,246 @@
+# Contributing to our Blog
+
+Authors who are informed and passionate about technology and IT write technical
+blogs that range from high-level, conceptual overviews to deep dives with
+code snippets. They all offer insight into solving problems and getting the most
+out of applications and platforms. Many also add a personal perspective to the
+conversation. These authors share their journeys, trials, and triumphs.
+
+First, thanks for your interest in contributing and helping us craft quality
+content for our official Technical Blog. Second, in order to make contributing
+a pleasant experience while maintaining a visual and consistent content standard,
+complete these steps before writing and submitting a blog entry for publishing.
+
+For ideas of topics to blog about, check out the end of this file!
+
+#### Preparing for your blog entry
+
+1. Fork this repo (https://github.com/rackerlabs/technical-blog), then
+   clone your fork.
+
+2. Before you start working, make sure your local content is up-to-date and
+   merged with the right upstream branch:
+
+      ```bash
+         git remote add upstream git@github.com:rackerlabs/docs-developer-blog.git
+         git checkout master
+         git fetch upstream
+         git checkout -b name-of-your-branch
+         git merge upstream/master
+      ```
+
+3. Create a new folder inside the `/hugo/content/blog` directory with the following naming
+   convention: `YYYY-MM-DD-title-of-your-post`, where `YYYY-MM-DD` is the
+   date you want your entry to be published.
+
+4. Within the newly created folder, create a new file with the following naming convention:
+   `index.md`.The published URL for your post becomes `https://docs.rackspace.com/blog/title-of-your-post/` when published.
+
+**Note:** If your post has images, you should place the image files within the newly created
+folder.
+
+#### Formatting your post
+
+The post should contain front-matter, an excerpt, and the actual content.
+
+##### Front-matter
+
+Add Jekyll front-matter (or metadata) to the top of the file you created in
+the previous step. for example:
+
+```
+---
+layout: post
+title: "Blog entry title"
+date: YYYY-MM-DD 23:59
+comments: true
+author: Author(s) name(s)
+published: true
+authorIsRacker: true
+#
+# The *authorAvatar* and *bio* entires are optional, but include them if you can!
+# The avatar must be a hosted image, such as a gravatar.
+#
+authorAvatar: 'https://www.gravatar.com/avatar/<insert hash for your headshot>'
+bio: "<insert a sentence or two about yourself in first or third person>"
+categories:
+    - This Category
+    - That Category
+    - Other Category
+#
+# Use canonical entry if you are republishing a blog from another site, such as
+# your personal blog.  Do  NOT republish without the autor's explicit permission.
```

Comment:
s/autor's/author's

Reply:
Didn't work on this section and it is now commented out but I corrected the typo anyways.